_templates/sidebar_links.html

```diff
@@ -10,27 +10,23 @@
   <script type="text/javascript">
     nbviewer_random_thumbnail();
   </script>
-	       
-</div> 
+
+</div>
 </div>
 
 <div class="sidebarblock">
 <h3>{{ _('Community') }}</h3>
 
   <ul class="simple">
-    <li><a class="reference external" href="https://gitter.im/ipython/ipython/help">
-            Help Chat Room</a></li>
-    <li><a class="reference external" href="http://stackoverflow.com/questions/tagged/ipython">
```

Comment:
I think I'd leave Stackoverflow at the top of this section - I'd rather people asked questions there by default, because I think third parties feel more comfortable helping on SO than on our mailing list.